spec/open_api_spec.rb

```diff
@@ -6,7 +6,7 @@
       next if r.engine? # Don't care right now...
       next if r.action == "invalid_url_error"
 
-      array << {:verb => r.verb, :path => r.path.split("(").first.sub(/:[_a-z]*id/, ":id")}
+      array << {:verb => r.verb, :path => r.path.split("(").first.gsub(/:[_a-z]*id/, ":id")}
```

Comment:
With the removal of the nested subcollection route can we bring this back to the original format?